### ISSUE TITLE
Please add -msse2 on i386 architectures

### ISSUE THREAD

**Issue by yurivict** [vxl/vxl#795]
vxl depends on SSE2 but it isn't enabled by default on i386.
See the failure log: http://beefy10.nyi.freebsd.org/data/113i386-default/549877/logs/vxl-2.0.2.629.log

**Comment by yurivict** [comment]
Actually, this should be done in the ITK project.

https://github.com/InsightSoftwareConsortium/ITK/issues/2011

**Comment by hjmjohnson** [comment]
Is this a duplicate of #761 ?

**Comment by yurivict** [comment]
I think so, this is because it has never been fixed.
